catalog/format/iceberg/src/main/java/org/projectnessie/catalog/formats/iceberg/meta/IcebergPartitionSpec.java

```diff
@@ -123,7 +123,7 @@ default boolean compatibleWith(IcebergPartitionSpec other) {
       IcebergPartitionField thisField = fields().get(i);
       IcebergPartitionField thatField = other.fields().get(i);
       if (thisField.sourceId() != thatField.sourceId()
-          || !thisField.transform().toString().equals(thatField.transform().toString())
+          || !thisField.transform().equals(thatField.transform())
```

Comment:
This does not look cosmetic to me :)

Reply:
`String.toString()` is :)

Reply:
Ah, ok... I only looked at the textual diff :)